feature/staticroute/tests/README.md

```diff
@@ -0,0 +1,35 @@
+# RT-1.25: Management network-instance default static route
+
+## Summary
+
+Validate static route functionality in Management network-instance (VRF).
+
+## Procedure
+
+
+*  Configure DUT with Management VRF and ATE1, ATE2 interfaces configured within this VRF
+*  Configure IPv4 and IPv6 default routes within Management VRF pointing to ATE2 interface
```

Comment:
Did we miss to mention what port in the DUT should be part of the VRF MGMT? There should be an interface in the same subnet as ATE2?

Reply:
line 10, specifies that the interfaces facing ate1 and ate2 from the dut, should be under the mgmt vrf.